### DISCUSSION TITLE
color painting not display ans not print when the shrinkage is not at 100% bug

### DISCUSSION THREAD

**Discussion by flodooby** [OrcaSlicer/OrcaSlicer#3906] [General]
Hello,

In orcaslicer I have a problem with color painting when the shrinkage is not at 100%.

* shrinkage is 100%, color painting is ok in preview and printing is ok

* shrinkage 99% 98%,95%, ...., all color painting are not displayed and the colors are not printed.

What am I not doing well?

Sorry for my language I'm French

![Bug color](https://github.com/SoftFever/OrcaSlicer/assets/118819450/aa0aa6c6-39c1-4b8c-8155-d99e728ce03c)

**Comment by rollallover** [comment]
Same problem with the modified shrinkage (Orca 1.9.0). Maybe it's not considered as a bug because different shrinkage may result into wall misfit. But I think it should work when one uses the same filament preset for all colors, hence with the same shrinkage.
